src/app/view/components/form-password/form-password.component.html

```diff
@@ -10,7 +10,5 @@
   >
     <mat-icon>{{ hide ? 'visibility_off' : 'visibility' }}</mat-icon>
   </button>
-  <mat-error *ngIf="pwdForm.controls['pwd'].hasError('required')">
-    La password è <strong>obbligatoria</strong>
-  </mat-error>
+  <mat-error *ngIf="pwdForm.controls['pwd'].hasError('required')">Password is <strong>required</strong></mat-error>
```

Comment:
direi di usare una stringa generica "this field is required" e riutilizzarla dappertutto invece che "password is required"

Reply:
E in che modo gioverebbe?

Reply:
è il concetto di cui ti parlavo in call un paio di sere fa, cioè il fatto che meno literals hai più l'app è gestibile (e se fosse necessario, traducibile usando angular i18n). 

metti che abbiamo 10 form diversi in cui c'è un campo email, su 5 di questi il testo è "email is required" su 2 è "this email is required", su uno è "this mail is required" e su altri 2 è "this field is required". stai dando all'utente un messaggio non consistente. è lo stesso principio del DRY.

Questa cosa la eviti a priori se utilizzi una costante (che poi può pure venire internazionalizzata)